information/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
 
     implementation group: "org.apache.commons", name: "commons-text", version: commons_text_version  // Apache 2.0
     implementation group: "org.apache.commons", name: "commons-collections4", version: commons_collections_version  // Apache 2.0
+    api group: "com.google.guava", name: "guava", version: guava_version
```

Comment:
Why is this necessary?

Reply:
This is used for the EvictingQueue to limit the debug information of the catalog.